### PR TITLE
chore(): PR template

### PR DESCRIPTION
**Pull request by ShaMan123**
Created a PR template with a notice regarding a PR lockdown for the migration.
Needless to say I shall be enforcing this lockdown on myself as of now 😐 (Well maybe just one more rtl fix).

😅

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<!--        
+        📣 IMPORTANT NOTICE - PR LOCKDOWN 🔒    04/2022
+        We are excited to announce that fabric is migrating to modern typescript/javascript 🤩.
+        This means we will ⛔ not be accepting any PRs out of scope with the migration.
+        We understand this might be annoying but wasted work is ever more so.
+        The migration will be extreme on the source code so PRs from before will probably become stale to the point of death after the migration.
+        It hurts us the throw away good work, effort and time put into fabric so please stay patient.
+        You are welcome to join the migration effort 🔨
+        https://github.com/fabricjs/fabric.js/issues/7596
```
Comment (asturur) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:29]: We can also say that if they have a small fix they can still PRO it to the v5.x branch. If is important or not already solved in the migration, can be ported after.

Comment (ShaMan123) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:29]: DONE [304dd2f](https://github.com/fabricjs/fabric.js/pull/7857/commits/304dd2f49d3ca82d150e0ec8f2d7cba4ccd31b94)

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Hi there!
+        Thanks for taking the time and putting the effort into making fabric better! 💖
+        Take a look at ./CONTRIBUTING.md for crucial instructions regarding local setup, testing etc.
+        https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md
+
+        Adding tests that verify your fix and safegurad it from unwanted loss and changes is a MUST.
+
+        PRing is not always simple, don't hestitate to ask for help (beware of 'em gotchas 😓).
+        We appreciate your effort and would like the process to be productive and enjoyable.
+        A strong community means a stong and better product for everyone.
```
[the remark this answers is not in the document]
Comment (ShaMan123) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: [304dd2f](https://github.com/fabricjs/fabric.js/pull/7857/commits/304dd2f49d3ca82d150e0ec8f2d7cba4ccd31b94)

**Review by asturur — APPROVED** [review]

**Comment by ShaMan123** [comment]
The template isn't visible when opening a PR
I probably named something wrong.
But what?

https://docs.github.com/en/enterprise-server@3.4/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

**Pull request merged**